EpikCord/client/event_handler.py

```diff
@@ -381,8 +381,7 @@ async def ready(self, data: dict):
 
                 for guild_id in command.guild_ids or []:
                     command_sorter[guild_id].append(command_payload)
-                else:
-                    command_sorter["global"].append(command_payload)
+                command_sorter["global"].append(command_payload)
```

Comment:
Isn't this incorrect? The command_sorter has a global key, for when the guild_ids is not specified. So won't this add them to both?

Reply:
the thing is that you are doing a
```py
for ...
   ...
else:
   ...
```
But since there is no `break` in the for, the `else` will be always executed

Reply:
And it won't add to both lists, right?

Reply:
it was already doing that  xD

Reply:
Oh what, welp, with this PR does that get fixed?

Reply:
Well that pr don't fix this, it just made the error more optimized lol

Reply:
I mean it helped you to find it, that a good thing

Reply:
Alright, fair enough